cli/package.json

```diff
@@ -52,6 +52,7 @@
     "babel-eslint": "^10.0.3",
     "babel-jest": "^25.1.0",
     "eslint": "^6.8.0",
+    "eslint-config-prettier": "^6.10.0",
     "eslint-plugin-flowtype": "^4.6.0",
     "eslint-plugin-prettier": "^3.1.2",
```

Comment:
Should this be removed now that `prettier` is no longer in `plugins`?

Reply:
yeah we could, `eslint-plugin-prettier`'s recommended config does extend it

Reply:
Oops, sorry I meant remove `eslint-plugin-prettier` but keep `eslint-config-prettier`.

Reply:
no we definitely need both if you want to check formatting via `eslint`

Reply:
I kinda blame `eslint` for introducing formatting lint rules without being a true code formatter, if they hadn't done that this would be way less confusing.